modules/storages/spec/features/delete_project_storage_and_file_links_spec.rb

```diff
@@ -46,7 +46,7 @@
   let(:file_link) { create(:file_link, storage:, container: work_package) }
   let(:second_file_link) { create(:file_link, container: work_package, storage:) }
   let(:delete_folder_url) do
-    "#{storage.host}/remote.php/dav/files/#{storage.username}/#{project_storage.project_folder_path.chop}"
+    "#{storage.host}/remote.php/dav/files/#{storage.username}/#{project_storage.project_folder_path.chop}/"
```

Comment:
@ba1ash isn't the `chop` removing the last slash? And now you add one again... Can't we simply remove both?